sql/rpl_info_table.cc

```diff
@@ -23,6 +23,13 @@
 #include "rpl_info_table.h"
 #include "rpl_utility.h"
 
+#ifdef WITH_WSREP
```

Comment:
Wouldn't it be better to go with a more object-oriented approach and create some RAII object for storing previous value and restoring it when going out of scope?
Edit: OK, I see it comes from upstream...